### PR TITLE
doc(changelog): create a new file

### PR DESCRIPTION
**Pull request by jimmyfortinx**
Signed-off-by: Jimmy Fortin <jimmy.fortin@corp.ovh.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Changelog
+
+## v2.0.0
+
+### Breaking change
+
+- Update `oui-navbar` styles and markup
+- New heigth for multiple components (40px):
```
Comment (neolitec) [inline CHANGELOG.md]: height

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Changelog
+
+## v2.0.0
+
+### Breaking change
```
Comment (neolitec) [inline CHANGELOG.md]: changes :)

Comment (jimmyfortinx) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (jimmyfortinx) [reply CHANGELOG.md]: Done

**Pull request merged**